src/back-end/lib/db.ts

```diff
@@ -344,11 +343,14 @@ export async function readManyAffiliations(connection: Connection, userId: Id):
 
 export async function readOneAffiliation(connection: Connection, user: Id, organization: Id): Promise<Affiliation> {
   const result = await connection('affiliations')
+    .join('organizations', 'affiliations.organization', '=', 'organizations.id')
+    .select('affiliations.*')
     .where({
-      user,
-      organization
+      'affiliations.user': user,
+      'affiliations.organization': organization
     })
-    .andWhereNot({ membershipStatus: MembershipStatus.Inactive })
+    .andWhereNot({ 'affiliations.membershipStatus': MembershipStatus.Inactive })
+    .andWhereNot({ 'organizations.active': false })
```

Comment:
I think you can combine these two `andWhereNot` calls into a single one by merging the argument objects.